src/PKSim.Core/Model/TransportDirection.cs

```diff
@@ -1,69 +1,190 @@
-         return transportDirection;
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxPlasmaToBloodCells;
+            default:
+               return EffluxBloodCellsToPlasma;
+         }
       }
 
-      public static IReadOnlyCollection<TransportDirection> All() => _allTransportDirections;
-   }
-   public class TransportDirection
-   {
-      public TransportDirectionId TransportDirectionId { get; }
-      public string DisplayName { get; }
-      public string Description { get; }
-      public ApplicationIcon Icon { get; }
-      public bool CanBeSetGlobally { get; }
+      public static TransportDirectionId DefaultMucosaDirectionFor(TransportType transportType)
+      {
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxLumenToMucosaIntracellular;
+            case TransportType.PgpLike:
+               return PgpMucosaIntracellularToLumen;
+            default:
+               return EffluxMucosaIntracellularToLumen;
+         }
+      }
+
+      public static TransportDirectionId DefaultTissueDirectionFor(TransportType transportType)
+      {
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxInterstitialToIntracellular;
+            case TransportType.PgpLike:
+               return PgpIntracellularToInterstitial;
+            default:
+               return EffluxIntracellularToInterstitial;
+         }
+      }
 
-      public TransportDirection(TransportDirectionId transportDirectionId, string displayName, string description, ApplicationIcon icon, bool canBeSetGlobally)
+      public static TransportDirectionId DefaultBrainTissueDirectionFor(TransportType transportType)
       {
-         TransportDirectionId = transportDirectionId;
-         DisplayName = displayName;
-         Description = description;
-         Icon = icon;
-         CanBeSetGlobally = canBeSetGlobally;
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxBrainInterstitialToTissue;
+            case TransportType.PgpLike:
+               return PgpBrainTissueToInterstitial;
+            default:
+               return EffluxBrainTissueToInterstitial;
+         }
       }
 
-      public override string ToString() => DisplayName;
+      public static TransportDirectionId DefaultBloodBrainBarrierDirectionFor(TransportType transportType)
+      {
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxBrainPlasmaToInterstitial;
+            case TransportType.PgpLike:
+               return PgpBrainInterstitialToPlasma;
+            default:
+               return EffluxBrainInterstitialToPlasma;
+         }
+      }
+
+      public static TransportDirectionId DefaultDirectionFor(TransportType transportType, TransporterExpressionContainer expressionContainer)
```

Comment:
I need to write some tests for this. This is by far the most complicated code aside from creating the transporter. Not too complicated in fact :)